tailwind.config.ts

```diff
@@ -1,18 +0,0 @@
-import config from '@rubriclab/tailwind-config'
-import {Config} from 'tailwindcss'
-
-const tailwindConfig = {
-	content: ['./app/**/*.tsx', './lib/**/*.tsx'],
-	presets: [config],
-	theme: {
-		extend: {
-			colors: {
-				panel: '#0A0A0A',
-				'panel-border': '#171717',
-				grey: '#555555'
-			}
-		}
-	}
-} satisfies Config
-
-export default tailwindConfig
```

Comment:
Was this removed intentionally?

Reply:
Yes, shadcn-ui automatically makes tailwind.config.js, so I just merged the content from both configs into one file -> tailwind.config.js